hb_profiles.py

```diff
@@ -131,12 +131,14 @@ def getToken(self):
         except:
             self.token = None
 
-    def formatProfile(self, val):
-        """
-        takes in a dictionary of parameter values and plugs them into the specified template
-        """
-        message = hb_config.message.format(**val).encode('utf-8')
-        return message
+    # NOTE: This function is no longer used for inviting, but is still
```

Comment:
Not sure I understand. We comment this out but know it's being used? Or do you mean it is used by other Teahouse functions we might want in the future?